### ISSUE TITLE
Dark color scheme is hard to read

### ISSUE THREAD

**Issue by XCanG** [DannyMoerkerke/whatpwacando.today#52]
There some potential issues with dark color scheme. Here is screenshots:
![image](https://github.com/DannyMoerkerke/whatpwacando.today/assets/4011523/a9e70ed0-db48-45a6-abef-8b0befab42af)
![image](https://github.com/DannyMoerkerke/whatpwacando.today/assets/4011523/cf5ab6d2-ffb7-43e5-b2e5-724f37356f6f)

Windows 10 with latest Chromium is used.

**Comment by DannyMoerkerke** [comment]
Can you please be more specific?

**Comment by XCanG** [comment]
Isn't it obvious? The contrast ratio is low, reading white on gray as well as black on gray is hurt. The background need to be darker.

Even in Github if you choose [Dark Dimmed](https://github.com/settings/appearance) theme it would have twice as more contrast comparing to colors on your site, and regular dark is even more distinguished.

**Comment by XCanG** [comment]
If you look at the contrast in Dev Tools you will see.
Dark theme:
![image](https://github.com/DannyMoerkerke/whatpwacando.today/assets/4011523/9451c65a-150b-493a-b42e-cf02808e29ad)

Dim theme:
![image](https://github.com/DannyMoerkerke/whatpwacando.today/assets/4011523/8955113d-790d-493e-bcce-043fc7306dda)

Your site is just slightly above threshold of 4.5
![image](https://github.com/DannyMoerkerke/whatpwacando.today/assets/4011523/0b1c76fa-1949-44d0-a023-edbc67eefa45)
If I just slightly shifted color down it will show warning:
![image](https://github.com/DannyMoerkerke/whatpwacando.today/assets/4011523/f85d424f-49bc-4d29-bf70-015bc0c6fd51)

**Comment by XCanG** [comment]
But the buttons and icons is even more below threshold:
![image](https://github.com/DannyMoerkerke/whatpwacando.today/assets/4011523/8a2fc29f-7b14-4654-8f46-05d6050edac0)
![image](https://github.com/DannyMoerkerke/whatpwacando.today/assets/4011523/313d8cd0-0d91-4106-9ada-6fe0ef59337d)

Some even unreadable, but because of the styles I can't focus on them _(disabled button may be an exception, but input label is not disabled)_:
![image](https://github.com/DannyMoerkerke/whatpwacando.today/assets/4011523/5a55d0d1-5ece-44c4-ae0e-3df1b67e9e57)

In any way, it was feels kinda strange, because the site has nice examples, but color choice was so bad. I had to squint my eyes to read that texts.

**Comment by DannyMoerkerke** [comment]
Thanks for the clarification! I will fix it asap.

**Comment by mertdy** [comment]
![Screenshot_2024-03-29-21-55-53-359_com.android.chrome-edit.jpg](https://github.com/DannyMoerkerke/whatpwacando.today/assets/24376224/1408454b-9e40-404d-bd4d-18d580e11d98)

I have the same problem on Android 14. My device is on Dark Mode.

**Comment by OffCrazyFreak** [comment]
Just change the main background color in root for prefers dark theme to #1e1e1e

![Image](https://github.com/user-attachments/assets/7c91fbcb-14fc-4043-b0fe-e942ae539175)

**Comment by dragontheory** [comment]
Very nice!

**Comment by DannyMoerkerke** [comment]
Just implemented #1e1e1e as background color.

**Comment by OffCrazyFreak** [comment]
Glad to be of help 😄 
I guess this issue can be Resolved #52 hehe
